README.md

```diff
@@ -89,6 +89,7 @@ Community meeting recordings: [IREE YouTube channel](https://www.youtube.com/@ir
 
 Date | Title | Recording | Slides
```

Comment:
nit: there are now two long titles that spill over onto a second line. Maybe we could shorten them a bit?

<img width="924" height="498" alt="Image" src="https://github.com/user-attachments/assets/00cbd9e3-0300-4005-a88c-6efa297f170b" />

Reply:
Eh, I wouldn't change just the title of your talk since the table still spills on `Unveiling the Inner Workings of IREE: An MLIR-Based Compiler for Diverse H/W` and that one is harder to rename (I'd also expand "H/W" to "hardware"...)

<img width="866" height="476" alt="Image" src="https://github.com/user-attachments/assets/ad7b2488-2bce-4078-ba2c-c914d15ab50f" />

Up to you :P